sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -188,11 +203,31 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     }
   }
 
+  // This will abort any pending request in the IdentityClient
+  abortRequests(): void {
+    let controller: AbortController | undefined;
+    while ((controller = this.abortControllers.shift())) {
+      controller.abort();
+    }
+  }
+
   sendGetRequestAsync<T>(
     url: string,
     options?: NetworkRequestOptions
   ): Promise<NetworkResponse<T>> {
-    const webResource = new WebResource(url, "GET", options?.body, {}, options?.headers);
+    const controller = new AbortController();
+    this.abortControllers.push(controller);
+
+    const webResource = new WebResource(
+      url,
+      "GET",
+      options?.body,
+      {},
+      options?.headers,
+      false,
+      false,
+      controller.signal
```

Comment:
MSAL has no way to pass abort signals, so I added a layer to control this independently of MSAL. The idea is that this can be improved over time, of course.

Something similar could be useful to send tracing spans too. I decided against that change because to do it right it would take considerable time, meaning I left tracing at the credentials methods level for now.